glitter-bug/Trestlebridge-Farms
# when added grazing farm, console writes

# Description

Changes made in CreateFacility.cs

Lines 21-23
 ```Console.WriteLine ("You created a new Grazing farm!");```
 ```Console.WriteLine ("Press Enter to go to the main Menu");```
 ```Console.ReadLine ();```

Fixes # (issue)

Ticket 1 : When the user creates a grazing farm, there is a message telling that the user created successfully.

## Type of change




# Testing Instructions

In the main menu, select the option 1, then create a grazing farm. The message will appear once you created the farm.

# Checklist:
